src/analytics/helpers.js

```diff
@@ -51,6 +51,7 @@ const supportedDimensions = [
   'filter_combination',
   'filter_type',
   'toggled_filter_item',
+  'shared_dataset_id',
```

Comment:
Same here, the `shared` prefix has been added to the shared dataset to differentiate it from the dataset in GA report.